client/src/reducers/auth.js

```diff
@@ -1,20 +1,21 @@
-import { LOGIN, REGISTER } from '../sagas/types';
+import { auth } from '../sagas/types';
+
+const actions = auth.reducer;
 
 const initialState = {
-  user: '',
-  displayName: '',
+  id: '',
+  name: '',
 };
 
 export default (state = initialState, { type, payload }) => {
   switch (type) {
-    case LOGIN:
-      console.log(`LOGIN: ${payload.username}`);
-      return { ...state, user: payload.username };
-    case REGISTER:
-      console.log(
-        `REGISTER: ${payload.username} with displayName = ${payload.displayName}`
-      );
+    case actions.LOGIN:
+    case actions.REGISTER:
```

Comment:
does js `fallthrough` by default? I miss golang here where this would be written as 

```go
case actions.LOGIN, actions.REGISTER:
```

instead. Does this have that functionality?

Reply:
Yes, it falls through. I struggled with which evil was better: copy pasting the `case` body or using the fallthrough to our advantage. What do you think?

Reply:
👍 use the fallthrough for js makes sense. DRY